FastTimeSeries/src/models/RNN.jl

```diff
@@ -24,5 +30,5 @@ function RNNModel(recbackbone,
                   dropout_rate = 0.0)
     
     dropout = dropout_rate == 0 ? identity : Dropout(dropout_rate)
-    Chain(recbackbone, dropout, finalclassifier)
-end
+    Chain(tabular2rnn, recbackbone, dropout, finalclassifier)
```

Comment:
I don't think we should be doing this dense to slices transform in the model itself. If you just need something RNN friendly, relying on the built-in support for dense inputs should be enough. The `permutedims` can stay for now, but even that probably shouldn't be in the gradient hot path (it allocates O(n) both ways).

Reply:
An alternative for now is to make the data pipeline spit out the vector of arrays. We can then revisit if/when you add models like CNNs which expect dense inputs.

Reply:
I think the "ideal" place to do this transform would be inside the training loop. I am not sure how to exactly do that for FastAI.jl. Is there a way ?

Since the second phase would involve using some CNNs, using data pipeline to spit out vector of arrays would not work.